src/App.js

```diff
@@ -1,10 +1,25 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, { useState } from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+  const [homeValue, setHomeValue] = useState(0);
+  const [awayValue, setAwayValue] = useState(0);
+
+  const [presentTime, setPresentTime] = useState(0);
+
+  const [color, setColor] = useState('teal');
+  const styles = {
+    backgroundColor: "#9ACD32"
+  }
+
+  function Both() {
```

Comment:
I would stay away from capitalizing the first letter of functions - that's reserved for classes and components (in react)